### DISCUSSION TITLE
Architectures supported

### DISCUSSION THREAD

**Discussion by upintheairsheep** [darlinghq/darling#1418] [General]
Question: what architectures of macOS are supported?
ARM64 (Apple Silicon apps)
x64 (macOS 64 Bit apps)
x86 (macOS 32 Bit apps)
PPC (macOS PowerPC apps)

**Comment by CuriousTommy** [comment]
Currently, x86 and x64 are support in Darling. There is on-going work to get Darling working on ARM64 Linux.

**Comment by upintheairsheep** [comment]
Is PPC support planned?

**Reply by CuriousTommy** [reply]
When we have a working Rosetta alternative, yes. I recommend reading this comment for more details: https://github.com/darlinghq/darling/issues/863#issuecomment-675834045
